### DISCUSSION TITLE
Text of the update message

### DISCUSSION THREAD

**Discussion by Tealk** [ImranR98/Obtainium#1644] [Q&A] [answered]
Hello,

I would like to know if it is intentional that the update message says that there may have been an update or is that a mistake in the translation?

German:
> xxx wurde möglicherweise aktuallisiert auf xxx

**Comment by DwainZwerg** [comment, marked as answer]
Nein, das ist kein Fehler. Es gibt zwei Meldungen (auch auf englisch). Einmal die App wurde aktualisiert (bei der nativen Methode):https://github.com/ImranR98/Obtainium/blob/2b27902d5f29d48f1c66b469c327d4ee7ea86958/assets/translations/de.json#L359 und einmal die App wurde möglicherweise installiert (bei der SUI/Shizuku-Methode):https://github.com/ImranR98/Obtainium/blob/2b27902d5f29d48f1c66b469c327d4ee7ea86958/assets/translations/de.json#L367

----
No, this is not an mistake. There are two messages (also in English). Once the app has been updated (for the native method):https://github.com/ImranR98/Obtainium/blob/2b27902d5f29d48f1c66b469c327d4ee7ea86958/assets/translations/de.json#L359 and once the app may have been installed (for the SUI/Shizuku method):https://github.com/ImranR98/Obtainium/blob/2b27902d5f29d48f1c66b469c327d4ee7ea86958/assets/translations/de.json#L367

**Reply by Tealk** [reply]
And what logic is used to select which message is used? Because the update is installed reliably.

**Reply by ImranR98** [reply]
It's reliable most of the time but it is possible for the update to fail, and when that happens there's no way for the app to know about it.

**Reply by ImranR98** [reply]
When Shizuku is used, the error does get caught and the user can be notified, that's why the wording of the success message is more confident.
